### DISCUSSION TITLE
关于直播间时间同步功能

### DISCUSSION THREAD

**Discussion by u2shana** [the1812/Bilibili-Evolved#2219] [功能建议 / Ideas]
https://github.com/the1812/Bilibili-Evolved/discussions/1342
以前有提及过这个功能，今天发现有人做出来了
![QQ截图20210826150109](https://user-images.githubusercontent.com/14146754/130916490-32df86f7-d352-467d-ac02-abccbaf7a574.jpg)
https://github.com/indefined/UserScripts/tree/master/bilibiliLive
不过这脚本其他功能对evolved兼容性不太好
作者对于这个功能的态度也挺适合v2的独立组件风格
https://github.com/indefined/UserScripts/issues/66#issuecomment-877543870
所以是否可以移植过来呢？

**Comment by the1812** [comment]
要等我移植的话得等 v2 正式版发布后了, 可以先让作者 Fork v2 写一个(
